internal/autoevent/executor.go

```diff
@@ -77,7 +77,12 @@ func (e *Executor) Run(ctx context.Context, wg *sync.WaitGroup, dic *di.Containe
 				if event.Origin == 0 {
 					event.Origin = common.GetUniqueOrigin()
 				}
-				go common.SendEvent(event, lc, container.CoredataEventClientFrom(dic.Get))
+
+				go func() {
```

Comment:
Please add comment describing `why` this is needed.

Reply:
Added comment.

Reply:
@lenny-intel do you think the comment is acceptable?

Reply:
@lenny-intel could you please review this PR again?